internal/log/middleware.go

```diff
@@ -164,7 +164,7 @@ func AccessHandler(f func(r *http.Request, status, size int, duration time.Durat
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			start := time.Now()
-			lw := NewWrapResponseWriter(w, 2)
+			lw := NewWrapResponseWriter(w, 1)
```

Comment:
```go 
// The protocol version for incoming server requests.
//
// For client requests, these fields are ignored. The HTTP
// client code always uses either HTTP/1.1 or HTTP/2.
// See the docs on Transport for details.
Proto      string // "HTTP/1.0"
ProtoMajor int    // 1
ProtoMinor int    // 0
```
```go
lw := NewWrapResponseWriter(w, r.ProtoMajor)
````

🤷‍♂ might work? Thoughts?

Reply:
Oh maybe, let me try that. I'll also play around with that you said in the above comment.